index.html

```diff
@@ -26,17 +26,17 @@ <h4>
     <ul>
       <li>
         <strong>
-          14 Jun 2024: The seL4 summit Program is available
-          </strong>
+          2 July 2024: new seL4 release available
```

Comment:
```suggestion
          2 July 2024: New seL4 release available
```

Just to be consistent with all the previous headlines.